src/CssParsing/StyleRule.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier\CssParsing;
+
+/**
+ * This class represents a CSS style rule, including a media query, selectors and a declarations block.
+ *
+ * @internal
+ */
+class StyleRule
+{
+    /**
+     * @var string
+     */
+    private $mediaQuery;
+
+    /**
+     * @var array<int, string>
+     */
+    private $selectors;
+
+    /**
+     * @var string
+     */
+    private $declarationsBlock;
```

Comment:
These can be replaced with a single [`DeclarationBlock`](https://github.com/sabberworm/PHP-CSS-Parser/blob/master/src/RuleSet/DeclarationBlock.php) property once #1015 is merged.

Reply:
Done.